src/app/components/mno-app-install-btn/mno-app-install-btn.coffee

```diff
@@ -172,7 +176,9 @@ angular.module 'mnoEnterpriseAngular'
             authorizedOrganizations = _.filter(currentUser.organizations, (org) ->
               MnoeOrganizations.role.atLeastAdmin(org.current_user_role)
             )
-            vm.canProvisionApp = !_.isEmpty(authorizedOrganizations)
+
+            organization = MnoeOrganizations.selected.organization
+            vm.canProvisionApp = atLeastAdmin(currentUser, organization)
```

Comment:
Rather than writing this helper function, we could just check that the selected organization is in authorizedOrganizations:
`vm.canProvisionApp = _.find(authorizedOrganizations, (org) -> org.id == organization.id)`

Reply:
Sounds better, will update, thanks.